### DISCUSSION TITLE
I'm a UX/UI designer

### DISCUSSION THREAD

**Discussion by Mahsa-eb47** [WeAreMahsaAmini/Core#31] [Introductions]
My name is Mahsa. I am a UX/UI designer. I have studied interactive media design in Canada, and I can help in designing the website and app. I was an animation director in Iran too, so I can make some simple short animations if needed.

**Comment by syavash** [comment]
Welcome, Mahsa jan 👋. We definitely can use some help in the Celebrity Wall (#18) project!

**Reply by anonymoper** [reply]
Nice to have you on the team! 

Please let me know what you think about this idea [https://github.com/WeAreMahsaAmini/Core/discussions/66](https://github.com/WeAreMahsaAmini/Core/discussions/66)
Thank you!
